webviz_subsurface/plugins/_multiple_regression.py

```diff
@@ -0,0 +1,1129 @@
+                    value=values[0],
+                    multi=False,
+                    clearable=False,
+                )
+            children.append(html.Div(children=[html.Label(col_name), selector,]))
+        return children
+
+    @property
+    def control_layout(self):
+        """ Layout to select e.g. iteration and response """
+        return [
+            html.Div(
+                [
+                    html.Div("Ensemble:", style={"font-weight": "bold"}),
+                    dcc.Dropdown(
+                        id=self.uuid("ensemble"),
+                        options=[{"label": ens, "value": ens} for ens in self.ensembles],
+                        clearable=False,
+                        value=self.ensembles[0],
+                        style={"marginBottom": "20px"},
+                    ),
+                ]
+            ),
+            html.Div(
+                [
+                    html.Div("Response:", style={"font-weight": "bold"}),
+                    dcc.Dropdown(
+                        id=self.uuid("responses"),
+                        options=[{"label": ens, "value": ens} for ens in self.responses],
+                        clearable=False,
+                        value=self.responses[0],
+                        style={"marginBottom": "20px"},
+                    ),
+                ]
+            ),
+            html.Div(
+                [
+                    html.Div(
+                        "Parameters:",
+                        style={
+                            "font-weight": "bold",
+                            "display": "inline-block",
+                            "margin-right": "10px",
+                        },
+                    ),
+                    dcc.RadioItems(
+                        id=self.uuid("exclude-include"),
+                        options=[
+                            {"label": "Exclusive", "value": "exc"},
+                            {"label": "Subset", "value": "inc"},
+                        ],
+                        value="exc",
+                        labelStyle={"display": "inline-block"},
+                        style={"fontSize": ".80em"},
+                    ),
+                    dcc.Dropdown(
+                        id=self.uuid("parameter-list"),
+                        options=[{"label": ens, "value": ens} for ens in self.parameters],
+                        clearable=True,
+                        multi=True,
+                        placeholder="",
+                        value=[],
+                        style={"marginBottom": "20px"},
+                    ),
+                ]
+            ),
+            html.Div("Filters:", style={"font-weight": "bold"}),
+            html.Div(children=self.filter_layout),
+            html.Div(
+                [
+                    html.Div(
+                        "Model settings:", style={"font-weight": "bold", "marginTop": "20px"},
```

Comment:
Maybe try to avoid claiming it is a "model".
Also: Think it could be nice to move this up a bit, e.g. above Parameters and Filters in the case that these sections are long (so that the Settings are not in the initial view).
```suggestion
                        "Settings:", style={"font-weight": "bold", "marginTop": "20px"},
```